### PR TITLE
Update ACK generation policy

### PR DESCRIPTION
**Pull request by adventureloop**
The discussion around the previous pull request #3501 seems to have stalled. We think a smaller change might also work.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -3261,8 +3261,14 @@ a max_ack_delay of 0 is used. The sender uses the receiver's `max_ack_delay`
 value in determining timeouts for timer-based retransmission, as detailed in
 Section 5.2.1 of {{QUIC-RECOVERY}}.
 
-An ACK frame SHOULD be generated for at least every second ack-eliciting packet.
-This recommendation is in keeping with standard practice for TCP {{?RFC5681}}.
+
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:3264]:
```suggestion
```

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -3261,8 +3261,14 @@ a max_ack_delay of 0 is used. The sender uses the receiver's `max_ack_delay`
...
+An ACK frame SHOULD be generated for at least every 10th ack-eliciting packet
+(the QUIC IW). A receiver could send an ACK frame at least every second 
+ack-elicitingpacket (recommended practice for TCP {{?RFC5681}}). QUIC permits
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:3267]:
```suggestion
ack-eliciting packet (recommended practice for TCP {{?RFC5681}}). QUIC permits 
```

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -3261,8 +3261,14 @@ a max_ack_delay of 0 is used. The sender uses the receiver's `max_ack_delay`
 value in determining timeouts for timer-based retransmission, as detailed in
 Section 5.2.1 of {{QUIC-RECOVERY}}.
 
-An ACK frame SHOULD be generated for at least every second ack-eliciting packet.
-This recommendation is in keeping with standard practice for TCP {{?RFC5681}}.
+
+An ACK frame SHOULD be generated for at least every 10th ack-eliciting packet
+(the QUIC IW). A receiver could send an ACK frame at least every second 
+ack-elicitingpacket (recommended practice for TCP {{?RFC5681}}). QUIC permits 
+a larger ratio between ACKs and data,which can reduce
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:3268]:
```suggestion
a larger ratio between ACKs and data, which can reduce
```

File: draft-ietf-quic-transport.md [hunk 4]
```diff
@@ -3261,8 +3261,14 @@ a max_ack_delay of 0 is used. The sender uses the receiver's `max_ack_delay`
...
+the number of ACK frames that are sent and need to be processed.
+The receiver can provide a higher rate of acknowledgements
+(e.g. every ack-eliciting packet) for the first 100 packets.
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:3271]:
```suggestion
(e.g., every ack-eliciting packet) for the first 100 packets.
```

**Comment by janaiyengar** [comment]
Closed by #3529.

**Pull request closed without merging**